### PR TITLE
standard section overhaul

### PR DESCRIPTION
**Pull request by fhildeb**
Forwarded changes from [PR166](https://github.com/lukso-network/docs/pull/166)

This PR for the standard section includes:

- resolving grammar and styling
- updating nav arrangement by not using decimals
- changing huge tables into HTML for readability and easier re-arrangement
- adding unified function and event declarations to smart contract descriptions
- adding/resolving external links
- changing headings on the menu 
     - without the hyphen as in links 
     - with spaces between the standard's words
       (spaceless writing should refer to functions or files)

### CODE REVIEW

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:70]: can you revert this pls

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:56]: no need to add "The function"

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:382]: i'm not sure if adding "the event" on every lines adds value here

@CJ42

File: docs/standards/smart-contracts/lsp1-universal-receiver-delegate-vault.md
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp1-universal-receiver-delegate-vault.md:6]: Please revert

The name should stay as it was before, it is the name of the smart contract

File: docs/standards/smart-contracts/lsp4-digital-asset-metadata.md
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp4-digital-asset-metadata.md:6]: please revert

File: docs/standards/smart-contracts/lsp6-key-manager.md
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp6-key-manager.md:6]: please revert - this is the contract name

File: docs/standards/smart-contracts/lsp6-key-manager.md
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp6-key-manager.md:53]: please remove "the function"

File: docs/standards/smart-contracts/lsp7-digital-asset.md
Comment (Hugoo) [inline docs/standards/smart-contracts/lsp7-digital-asset.md:6]: please revert

File: docs/standards/smart-contracts/01-introduction.md
Comment (Hugoo) [inline docs/standards/smart-contracts/01-introduction.md:10]: let's keep the package name in the link

**Review by Hugoo — CHANGES_REQUESTED** [review]
Nice !

Just a few things:

- Let's keep the `-` in titles for the standard
- Let's keep the contract name for the "implementation" pages + also add the contract name in the sidebar so it is consistent (`LSP6KeyManager` both in the title and in the sidebar)
- No need to add "The function..." - this is more like a "recipe" as it is for the API pages.

File: docs/guides/universal-profile/create-profile.md
Comment (fhildeb) [reply docs/guides/universal-profile/create-profile.md:40]: always e.g. I missed this one due to the underscore

File: docs/standards/faq/channel-nonce.md
Comment (fhildeb) [reply docs/standards/faq/channel-nonce.md:6]: See FAQ section on docs.lukso.tech. It uses the same structure

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:42]: Please re-add the previous links to "./interface-ids.md", as we need these links so that people can easily find the `bytes4` interface IDs of the _lsp-smart-contracts_

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:56]: The sentence descriptions in the contracts API docs need to be very short.
See Uniswap Contract API docs as an example for the sentence: https://docs.uniswap.org/protocol/reference/core/UniswapV3Factory#setowner

```suggestion
Returns the address of the current account owner.
```

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:101]:
```suggestion
Call any other smart contracts, transfer value, or deploy a new smart contract.
```

File: docs/standards/nft-2.0/01-introduction.md
Comment (fhildeb) [reply docs/standards/nft-2.0/01-introduction.md:10]: it is one text section, more space in between looks off sometimes

File: docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md:60]: I think this should be reverted, as hooks are not connected to contracts, but it's more something independant.
@YamenMerhi what do you think?

File: docs/standards/nft-2.0/04-LSP8-Identifiable-Digital-Asset.md
Comment (fhildeb) [reply docs/standards/nft-2.0/04-LSP8-Identifiable-Digital-Asset.md:24]: Good catch! Did not notice those leftover statics

File: docs/standards/standard-detection.md
Comment (CJ42) [inline docs/standards/standard-detection.md:28]: Please revert to keep it generic, as the functions can be called from Dapps interfaces (frontend + backend) and other smart contracts.

File: docs/standards/universal-profile/01-lsp0-erc725account.md
Comment (CJ42) [inline docs/standards/universal-profile/01-lsp0-erc725account.md:16]: Please revert here too.

File: docs/standards/universal-profile/01-lsp0-erc725account.md
Comment (CJ42) [inline docs/standards/universal-profile/01-lsp0-erc725account.md:18]: Please revert here too to keep it generic too (as these blockchain based accounts can be created by both EOAs and other contracts).

File: docs/standards/universal-profile/01-lsp0-erc725account.md
Comment (CJ42) [inline docs/standards/universal-profile/01-lsp0-erc725account.md:50]: Please keep the tables as markdown, as we agreed to not use HTML tables in docs.

**Review by CJ42 — COMMENTED** [review]
added comments

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (fhildeb) [reply docs/standards/smart-contracts/lsp0-erc725-account.md:18]: b) sounds nice. thank you!

File: docs/standards/smart-contracts/01-introduction.md
Comment (fhildeb) [reply docs/standards/smart-contracts/01-introduction.md]: good catch

File: docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md
Comment (fhildeb) [reply docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md:60]: reverted to "The token hooks of the contract. "

File: docs/standards/universal-profile/06-lsp5-received-assets.md
Comment (Hugoo) [inline docs/standards/universal-profile/06-lsp5-received-assets.md:3]: says 5 but filename says 6 - i suggest to remove the `06` from the filename

**Review by Hugoo — APPROVED** [review]
thanks for all these improvements !

Let's wait for @CJ42 review for the smart contract part and merge after his approval

File: docs/standards/smart-contracts/lsp1-universal-receiver-delegate-vault.md
Comment (fhildeb) [reply docs/standards/smart-contracts/lsp1-universal-receiver-delegate-vault.md:17]: Yes, they all should be the same

File: docs/standards/universal-profile/01-lsp0-erc725account.md
Comment (fhildeb) [reply docs/standards/universal-profile/01-lsp0-erc725account.md:3]: will do structure in separate PR

File: docs/standards/universal-profile/04-lsp6-key-manager.md
Comment (fhildeb) [reply docs/standards/universal-profile/04-lsp6-key-manager.md:297]: I don't know, I tried to get rid of most of these things. Missed that one

File: docs/standards/universal-profile/06-lsp5-received-assets.md
Comment (fhildeb) [reply docs/standards/universal-profile/06-lsp5-received-assets.md:3]: as above- will tackle structure in separate PR as it changes all the sidelinks

**Comment by fhildeb** [comment]
Explanation of using array[ &nbsp;] instead of array[ ]:
Arrays are declared with spaces between brackets to improve clarity, but if you have more than one possible type or a small device, the second bracket gets split up with a line break. That's why we need to use the space symbol to always keep it as one object.

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:143]: Suggests **bytes** -> `bytes`

as it is a parameter name.

```suggestion
Sets data in the account storage for a particular key as `bytes`.
```

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:155]: Same comment about "you" as mentioned above.

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:164]: Can we please always use present tense / 3rd person singular for the API docs.

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp0-erc725-account.md:208]: I find this style and sentence perfect 👍 
This should be used as a documented example for the conventions in the docs!

File: docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md:39]: Suggested change to stick to the convention of function parameters with backticks.

```suggestion
Registers the [**LSP1UniversalReceiverDelegate** `interfaceId`](./interface-ids.md)**.
```

File: docs/standards/smart-contracts/lsp4-digital-asset-metadata.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp4-digital-asset-metadata.md:46]: What about using a bullet list to describe the constructor here?

```markdown
Sets the **initial owner** of the contract, and the following data keys on the **[ERC725Y Key-Value Store](./lsp0-erc725-account#setdata)** of the contract:

- `name_`: token's name.
- `symbol_`: token's symbol.
```

File: docs/standards/smart-contracts/lsp6-key-manager.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp6-key-manager.md:103]: Nicely used callout 👏  ℹ️

File: docs/standards/smart-contracts/lsp9-vault.md
Comment (CJ42) [inline docs/standards/smart-contracts/lsp9-vault.md:85]: same comment as above

**Review by CJ42 — COMMENTED** [review]
Added comments.

There are some great sentences fixes, especially the description of the `authorizeOperator(...)`


Before -> Makes `operator` address an operator of `tokenId`.
After -> Sets the `operator` address as an operator of a particular `tokenId`.

File: docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md
Comment (YamenMerhi) [inline docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md:18]: I would prefer to keep the old one with fixing the grammar mistakes because this is not right:
> the vaults owned from the **[LSP9-Vault Standard](./lsp9-vault.md)** into your account storage

The vaults are not owned from the LSP9-Vault, they are owned by the account and their type is LSP9-Vault.

File: docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md
Comment (YamenMerhi) [inline docs/standards/smart-contracts/lsp1-universal-receiver-delegate-up.md:16]: The same remarks of the vault go here, the assets are not received from the lsp7/lsp8 standards.

**Review by YamenMerhi — COMMENTED** [review]
Cool!

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (Hugoo) [reply docs/standards/smart-contracts/lsp0-erc725-account.md:103]: agree

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (Hugoo) [reply docs/standards/smart-contracts/lsp0-erc725-account.md:124]: 1) agree with not using you. Maybe something generic like "The address to interact with".
2) field -> either address OR "the parameter"

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (Hugoo) [reply docs/standards/smart-contracts/lsp0-erc725-account.md:164]: agree

File: docs/standards/smart-contracts/lsp0-erc725-account.md
Comment (fhildeb) [reply docs/standards/smart-contracts/lsp0-erc725-account.md:208]: thank you!

File: docs/standards/smart-contracts/lsp4-digital-asset-metadata.md
Comment (fhildeb) [reply docs/standards/smart-contracts/lsp4-digital-asset-metadata.md:46]: Yes indeed! bullet list would be ideal

Comment (fhildeb) [reply docs/standards/smart-contracts/lsp4-digital-asset-metadata.md:46]: Should also use the same structure for LSP7 and LSP8 within the same folder

**Review by Hugoo — APPROVED** [review]

**Pull request merged**
